src/query.js

```diff
@@ -1,335 +1,708 @@
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function Modulo() {
+  return new Expr({ modulo: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function lt(...values) {
-  return {lt: varargs(values)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function LT() {
+  return new Expr({ lt: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function lte(...values) {
-  return {lte: varargs(values)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function LTE() {
+  return new Expr({ lte: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function gt(...values) {
-  return {gt: varargs(values)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function GT() {
+  return new Expr({ gt: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function gte(...values) {
-  return {gte: varargs(values)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function GTE() {
+  return new Expr({ gte: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function and(...booleans) {
-  return {and: varargs(booleans)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function And() {
+  return new Expr({ and: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function or(...booleans) {
-  return {or: varargs(booleans)}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {...module:query~ExprArg} terms
+ * @return {Expr}
+ */
+function Or() {
+  return new Expr({ or: Expr.wrap(varargs(arguments)) });
 }
 
-/** See the [docs](https://faunadb.com/documentation/queries#misc_functions). */
-export function not(boolean) {
-  return {not: boolean}
+/**
+ * See the [docs](https://faunadb.com/documentation/queries#misc_functions).
+ *
+ * @param {module:query~ExprArg} boolean
+ * @return {Expr}
+ */
+function Not(boolean) {
+  return new Expr({ not: Expr.wrap(boolean) });
 }
 
 // Helpers
 
 /** Adds optional parameters to the query. */
 function params(mainParams, optionalParams) {
```

Comment:
what do you mean hidden? from the jsdocs? the jsdocs by default don't expose functions that aren't exported